README.md

```diff
@@ -81,6 +90,7 @@
 
 <!-- > Add a link to your deployed project.-->
 - [Live Demo Link](https://miltonhenschel.github.io/Conference-Page/)
+- [Video presentation](https://www.loom.com/share/ea7190471a844c69bfe0f44099ecec13)
```

Comment:
You mentioned that creating the speaker's page dynamically makes the page load faster. On the contrary, it makes the code slower but makes updating your list of speakers faster.

Reply:
Oh! you watched my video presentation? Thanks for offering your remarks.

Reply:
Oh! you watched my video presentation? Thanks for offering your remarks.

Reply:
Thanks for the remarks @fobadara.

Reply:
Thanks for the remarks @fobadara.